internal/test/test.go

```diff
@@ -101,13 +139,26 @@ func (r *TestResult) String() string {
 	var b bytes.Buffer
 	writer := util.CreateTabWriter(&b)
 
-	_, _ = fmt.Fprintf(writer, cdcTests.PrettyPrintResults(r.Results))
+	for scriptPath, testResult := range r.Results {
+		_, _ = fmt.Fprint(writer, cdcTests.PrettyPrintResults(testResult, scriptPath))
+	}
+	if r.CoverageReport != nil {
+		_, _ = fmt.Fprint(writer, r.CoverageReport.CoveredStatementsPercentage())
+	}
 
 	_ = writer.Flush()
 
 	return b.String()
 }
 
 func (r *TestResult) Oneliner() string {
-	return cdcTests.PrettyPrintResults(r.Results)
+	message := ""
+	for scriptPath, testResult := range r.Results {
+		message += cdcTests.PrettyPrintResults(testResult, scriptPath)
+	}
+	if r.CoverageReport != nil {
+		message += r.CoverageReport.CoveredStatementsPercentage()
+		message += "\n"
+	}
```

Comment:
Maybe use a `strings.Builder` here

Reply:
Nice! Fixed in https://github.com/onflow/flow-cli/pull/827/commits/bf43d094fd0dc4ff2fe81f51cd4bf3126dfabcb8